src/hardware/sblaster.cpp

```diff
@@ -2180,13 +2180,38 @@ OplMode find_oplmode()
 		case SBT_PRO1: opl_mode = OplMode::DualOpl2; break;
 		case SBT_2:
 		case SBT_1: opl_mode = OplMode::Opl2; break;
-		case SBT_GB: opl_mode = OplMode::Cms; break;
+		case SBT_GB: opl_mode = OplMode::None; break;
 		case SBT_NONE: opl_mode = OplMode::None; break;
 		}
 	}
 	return opl_mode;
 }
 
+bool find_cms() {
+	const auto sect = static_cast<Section_prop*>(control->GetSection("sblaster"));
+	assert(sect);
+
+    // Backward compatibility with existing configurations
+	if (sect->Get_string("oplmode") == "cms") {
+		LOG_WARNING("The use of oplmode 'cms' is now deprecated.");
```

Comment:
Prefix the warning with the SB model like all the other log statements in this file and suggest a resolution. E.g.,

```cpp
LOG_WARNING("%s: The 'cms' setting for 'oplmode' is deprecated; please use `cms = on` instead.", CardType());
```